llvm/test/MC/WebAssembly/branch-hints-custom-high-low-thresholds.ll

```diff
@@ -0,0 +1,79 @@
+; RUN: llc -mcpu=mvp -filetype=obj %s -mattr=+branch-hinting -wasm-branch-prob-high=0x60000000 -wasm-branch-prob-low=0x0 -o - | obj2yaml | FileCheck %s
```

Comment:
Should test different values (multiple run lines, different check prefixes for FileCheck) to make sure the options actually work.